lang/po/Firmware_cs.po

```diff
@@ -1678,22 +1701,24 @@ msgstr ""
 #: ../../Firmware/mmu2/errors_list.h:200 ../../Firmware/mmu2/errors_list.h:240
 msgid "Pulley motor stalled. Ensure the pulley can move and check the wiring."
 msgstr ""
+"Motor idleru se zastavil. Ujistete se, ze neni pohyb blokovan a zkontrolujte "
```

Comment:
@DRracer is it Kladka motor?

Reply:
nope, Kladka means Pulley ... but this is obviously a copy-paste error

Reply:
is then "Motor kladce se zastavil.
correct?

Reply:
Deepl translation
`Kladkový motor se zastavil. Zkontrolujte, zda se kladka může pohybovat, a zkontrolujte zapojení.`